Add info about bonded interfaces in PXE provisioning (#3420)
theforeman/foreman-documentation: guides/common/modules/con_using-pxe-to-provision-hosts.adoc

For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
endif::[]

.Bonded network interfaces
You can configure a bonded interface that {Project} will use during the installation process, for example, to download installation content.
After provisioning completes, the provisioned system can also use the bonded interface.

> {Project} cannot PXE boot a bonded interface that requires configuration on a network switch as well as on your host.

Author: ```suggestion

[IMPORTANT]
====
{Project} cannot PXE boot a bonded interface that requires configuration on a network switch as well as on your host.
====

```

> When your host loads a kernel, either the kernel of an installer or an operating system, bonding works as expected.

Author: ```suggestion
After your host loads the kernel of an installer or the kernel of an operating system, bonding works as expected.
```

For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
endif::[]

.Bonded network interfaces
You can configure a bonded interface that {Project} will use during the installation process, for example, to download installation content.
After provisioning completes, the provisioned system can also use the bonded interface.

[IMPORTANT]
====
{Project} cannot PXE boot a bonded interface that requires configuration on a network switch as well as on your host.

> ====
> 
> After your host loads the kernel of an installer or the kernel of an operating system, bonding works as expected.
> Therefore, you can use a boot disk to work around PXE boot limitations when your bonded interface requires configuration on both a switch and your host.

Author: ```suggestion

After your host loads the kernel of an installer or the kernel of an operating system, bonding works as expected.
Therefore, you can use a boot disk to work around PXE boot limitations when your bonded interface requires configuration on both a switch and your host.
====
```

For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
endif::[]

.Bonded network interfaces

> You can configure a bonded interface on your host and use it for provisioning with PXE boot.

Reviewer: `use it for provisioning with PXE boot.` -> `use it during PXE based provisioning, after the actual PXE boot.`

> You can configure a bonded interface on your host and use it during PXE based provisioning after the actual PXE boot.

Reviewer 2: Maybe to specify it a bit more explicitly:
`You can configure a bonded interface to be used during the installation process (for downloading installation sources e.t.c.) and for the provisioned system (to be used after the provisioning process is finished).`

Author: ```suggestion
You can configure a bonded interface to be used during the installation process, such as for downloading installation content, and for the provisioned system to be used after the provisioning process is finished.
```

> You can configure a bonded interface to be used during the installation process, such as for downloading installation content, and for the provisioned system to be used after the provisioning process is finished.

Reviewer 3: ```suggestion
You can configure a bonded interface to be used during the installation process, such as for downloading installation content and for the provisioned system to be used after the provisioning process is finished.
```

> You can configure a bonded interface to be used during the installation process, such as for downloading installation content, and for the provisioned system to be used after the provisioning process is finished.

Author: I read it the same way, however, I'd suggest an alternative:
```suggestion
You can configure a bonded interface that {Project} will use during the installation process, for example, to download installation content.
After provisioning completes, the provisioned system can also use the bonded interface.
```

{Project} cannot PXE boot a bonded interface that requires configuration on a network switch as well as on your host.

> Once your host loads a kernel, either the kernel of an installer or an operating system, bonding works as expected.

Reviewer 3: ```suggestion
Bonding will work correctly once the host starts up with a kernel, whether it is from an installer or an operating system.
```

Author: I'm not going to use this suggestion because it introduces a future tense, a phrasal verb and pronoun, which all are undesirable for our style. However, I think that we shouldn't use "once" either.
```suggestion
When your host loads a kernel, either the kernel of an installer or an operating system, bonding works as expected.
```

> Therefore, if your bonded interface requires configuration on both a switch and your host, you can use a boot disk to circumvent limitations of such bonding in PXE boot.

Reviewer 3: ```suggestion
Therefore, you can use a boot disk to work around PXE boot limitations when your bonded interface requires configuration on both a switch and your host.
```
